Libplanet.Tests/TestUtils.cs

```diff
@@ -300,12 +300,17 @@ public static void AssertBencodexEqual(IValue expected, IValue actual)
             );
         }
 
-        public static void AssertSorted<T>(IEnumerable<T> list)
+        public static void AssertSorted<T>(IEnumerable<T> expected)
             where T : IComparable<T>
         {
-            Assert.True(list
-                .Zip(list.Skip(1))
-                .All(pair => pair.First.CompareTo(pair.Second) <= 0));
+            List<T> list = expected.ToList();
+            var ret = true;
+            for (int i = 0; ret && i < list.Count - 1; i++)
+            {
+                ret &= list[i].CompareTo(list[i + 1]) <= 0;
+            }
+
+            Assert.True(ret, "Given list is not sorted.");
```

Comment:
I know I originally implemented this, but if possible, it'd be also better to tell which indices are not in order.

Reply:
```suggestion
            foreach ((int Index, (var First, var Second)) in Enumerable
                .Range(0, list.Count() - 1)
                .Select(index => (index, (list[index], list[index + 1]))))
            {
                string errorMessage =
                    $"Given list is not sorted since item {First} at {Index} " +
                    $"is greater than item {Second} at {Index + 1}";
                Assert.True(First.CompareTo(Second) <= 0, errorMessage);
            }
```

Maybe something like this? 😗

Reply:
Amended at https://github.com/planetarium/libplanet/pull/2532/commits/664941f54ed04cdcb87adf7144da1643c93f0c78